README.md

```diff
@@ -526,6 +526,16 @@ Contributions of additional uses cases on top of ECS are welcome.
 
 
 
+# <a name="reserved-names"></a>Reserved Section Names
+
+ECS does not define the following field sets yet, but the following are expected
+in the future. Please avoid using them:
+
+- `match.*`
```

Comment:
This one is new to me :-)

Reply:
It's the result of vulnerability or threat detection. That's how the "match" is recorded (e.g. what field name matched, what time the detection occurred, etc.)